test/parallel/test-http2-large-write-destroy.js

```diff
@@ -32,6 +32,7 @@ server.listen(0, common.mustCall(() => {
 
   const req = client.request({ ':path': '/' });
   req.end();
+  req.resume();
```

Comment:
Feel free to leave a comment here that explains why this line is being added, so that nobody gets the idea to remove it because the test might pass without it :)

Reply:
Done!